megamek/src/megamek/client/ui/swing/boardview/StepSprite.java

```diff
@@ -63,84 +78,61 @@ public void prepare() {
         Image tempImage = new BufferedImage(BoardView1.HEX_W, BoardView1.HEX_H,
                 BufferedImage.TYPE_INT_ARGB);
         Graphics graph = tempImage.getGraphics();
+        Graphics2D g2D = (Graphics2D)graph;
 
-        if (GUIPreferences.getInstance().getAntiAliasing()) {
-            ((Graphics2D) graph).setRenderingHint(
-                    RenderingHints.KEY_ANTIALIASING,
-                    RenderingHints.VALUE_ANTIALIAS_ON);
-        }
+        GUIPreferences.AntiAliasifSet(graph);
 
         // fill with key color
         graph.setColor(new Color(0, 0, 0, 0));
         graph.fillRect(0, 0, BoardView1.HEX_W, BoardView1.HEX_H);
 
         // setup some variables
-        final Point stepPos = bv.getHexLocation(step.getPosition());
-        stepPos.translate(-bounds.x, -bounds.y);
-
```

Comment:
A rather roundabout way to calculate Point(0, 0)

Reply:
I believe I added this comment a few lines below that "//when is stepPos ever <> 0?" a few years ago. But I was afraid I was missing something...